09_regexp.md

```diff
@@ -986,71 +696,49 @@ while (match = number.exec(input)) {
 
 {{index "while loop", ["= operator", "as expression"], [binding, "as state"]}}
 
-This makes use of the fact that the value of an ((assignment))
-expression (`=`) is the assigned value. So by using `match =
-number.exec(input)` as the condition in the `while` statement, we
-perform the match at the start of each iteration, save its result in a
-binding, and stop looping when no more matches are found.
+以上示例巧妙地利用了((赋值))表达式 (`=`) 的值。所以一旦 `match = number.exec(input)` 成为 `while` 循环语句的条件，我们便可以在每一次迭代之初进行匹配，将匹配结果绑定，然后在找不到更多匹配项时停止循环。  
 
 {{id ini}}
-## Parsing an INI file
+## 解析一个 INI 文件
 
 {{index comment, "file format", "enemies example", "INI file"}}
 
-To conclude the chapter, we'll look at a problem that calls for
-((regular expression))s. Imagine we are writing a program to
-automatically collect information about our enemies from the
-((Internet)). (We will not actually write that program here, just the
-part that reads the ((configuration)) file. Sorry.) The configuration
-file looks like this:
+现在，我们来试图解决一个需要调用((正则表达式))的问题来总结这一章的内容。试想一下，我们现在打算写一个可以程序，它可以自动收集我们在((互联网))上仇敌的信息。(开玩笑，我们并不会真的在这里写这样一个程序，只是写其中可以读取((配置))文件的部分而已。)以下是我们的配置文件：
 
 ```{lang: "text/plain"}
 searchengine=https://duckduckgo.com/?q=$1
 spitefulness=9.7
 
-; comments are preceded by a semicolon...
-; each section concerns an individual enemy
+; 分号后面的是注释
+; 每一个区块针对一个仇敌
 [larry]
-fullname=Larry Doe
-type=kindergarten bully
+fullname=Larry Doe 拉里·兜
```

Comment:
i would have comments in a new line preceded by `;` to avoid confusion

alternatively, i'm not sure if these are worth translating (since it's code)? maybe a comment at the very beginning explaining the format or something?

Reply:
You are right. I will delete the translation.